system/services/presideObjects/PresideObjectService.cfc

```diff
@@ -1634,6 +1634,41 @@ component displayName="Preside Object Service" {
 		return getObjectAttribute( arguments.objectName, "dateModifiedField", "dateModified" );
 	}
 
+	/**
+	 * Returns the flag field name of the object
+	 *
+	 * @autodoc    true
+	 * @objectName Name of the object whose flag field you wish to get
+	 */
+	public string function getFlagField( required string objectName ) {
+		var flagEnabled = getObjectAttribute( arguments.objectName, "flagEnabled", "" );
+
+		if ( IsBoolean( flagEnabled ) && flagEnabled ) {
+			return getObjectAttribute( arguments.objectName, "flagField", "recordFlagged" );
+		}
+
+		return "";
+	}
+
```

Comment:
I suggest adding a helper here: `isFlaggingEnabled( objectName )` - this should be used over `Len( getFlagField() )` as a more explicit and readable check in handlers, etc.

Reply:
added